draft-ietf-intarea-provisioning-domains.md

```diff
@@ -840,6 +840,58 @@ Note: the above examples assume that the router has received its PvD IDs from up
 or via some other configuration mechanism. Another document could define ways for the router
 to generate its own PvD IDs to allow the above scenario in the absence of PvD ID provisioning.
 
+## Providing Additional Information to PvD-Aware Hosts
+
+In this example, the router indicates that it provides additional information using the H-flag.
+The Sequence Number on the PvD Option is set to 7 in this example.
+
+* RA Header: router lifetime = 6000
+* Prefix Information Option: length = 4, prefix = 2001:db8:cafe::/64
+* Recursive DNS Server Option: length = 3, addresses= \[2001:db8:cafe::53\]
+* PvD Option header: length = 3 + 5 + 4 , PvD ID FQDN = cafe.example.com.,
```

Comment:
Are you sure about the `3 + 5 + 4` length ?
I am not sure to understand how you computed it.

Reply:
Copy and paste error =)

Reply:
Fixed!